instrumentation/httpd/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+cmake_minimum_required(VERSION 3.12)
+
+project(opentelemetry-httpd)
+
+find_package(opentelemetry-cpp REQUIRED)
+find_package(Threads REQUIRED)
+find_package(protobuf REQUIRED)
+find_package(gRPC REQUIRED)
+find_package(CURL REQUIRED)
```

Comment:
Is curl needed?

Reply:
Yes without it I've got an error: `... links to target "CURL::libcurl" but the target was not found.`